### PR TITLE
add script for pulling SEP content from redcap

### PR DESCRIPTION
**Pull request by yatharthranjan**
Allows either:
1. pulling instrument metadata from REDCap API or 
2. Reading the CSV files of exported instruments from REDCap

TODO:

- [x] Add usage instructions in the README

Closes https://github.com/RADAR-base/hydra-kratos/issues/47 and https://github.com/RADAR-base/hydra-kratos/issues/46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
...
+
+The definitions are used by the self enrolment portal to render the [UI](https://github.com/RADAR-base/radar-self-enrolment-ui) and validate the user input. The definitions are written in JSON format and are loaded by the portal at runtime.
+
+## Definitions
+
+The definitions are divided into multiple files based on the type of information they define. The definitions are stored in the per-project directories.
+
+#### Study Information
+
+The study information definitions define the study information that is displayed to the user. The study information is displayed on the home page of the portal. The study information definitions are stored in the `study_info` directory.
```
Comment (afolarin) [inline README.md]:
```suggestion
The study information definitions define the study information displayed to the user on the portal's study home page. The study information definitions are stored in the `study_info` directory.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,70 @@
...
+
+#### Eligibility
+
+The eligibility definitions define the eligibility criteria that the user must meet to be able to participate in the study. The eligibility definitions are stored in the `eligibility` directory.
+
+#### Consent
+
+The consent definitions define the consent form that the user must agree to before participating in the study. The consent definitions are stored in the `consent` directory.
```
Comment (afolarin) [inline README.md]:
```suggestion
The consent definitions define mandatory and optional consent questions that the user must agree to before participating in the study. The consent definitions are stored in the `consent` directory.
```

**Comment by afolarin** [comment]
Can only see `Study Info`, `Eligibility`, `Consent`. Are the identifiable questionnaires also planned to be defined here?

**Comment by yatharthranjan** [comment]
We still have to decide that. Since they are questionnaires, it might be better to keep them in the aRMT definitions repo. However, since they are specific to SEP, they can also be stored here. I think they would be better in aRMT definitions as they are not project-specific.

**Review by afolarin — DISMISSED** [review]
LGTM

**Pull request merged**